sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/main/java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobCheckpointStore.java

```diff
@@ -193,7 +193,7 @@ public Flux<PartitionOwnership> claimOwnership(List<PartitionOwnership> requeste
                             LOGGER.atVerbose()
                                 .addKeyValue(PARTITION_ID_LOG_KEY, partitionId)
                                 .log(Messages.CLAIM_ERROR, error);
-                            return Mono.empty();
+                            return Mono.error(error);
```

Comment:
Hi @conniey,  wondering if we should be inspecting the error to decide whether to stop LB entirely or notify the error but continue. For example, looking at the error handling in .NET check point blob store https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/BlobCheckpointStore/BlobCheckpointStoreInternal.cs#L247-L265

Reply:
By notify of the error, you mean calling the user's callback rather than just logging it?

Reply:
We would use the existing processError() method for the callback

Reply:
All processor errors, whether it is from partition processing or from load balancing, should be sent to the `processError` callback that the user configures when creating the EventProcessorClient. The `errorContext` may not have partition details if the error is related to load balancing. 

The user then has a chance to inspect the error and decide whether to continue or stop the processor.